templates/src/components/App/App.js

```diff
@@ -50,20 +51,24 @@ class App extends React.PureComponent {
 
   render() {
     return (
-      <Fragment>
+      <>
```

Comment:
Not so sure about using fragments as `<>`, do u know how the state of tooling is with this?
> Note that many tools don’t support it yet so you might want to explicitly write <React.Fragment> until the tooling catches up.

**Ref:** https://reactjs.org/docs/fragments.html#short-syntax